experimental/PropertyDDS/packages/property-dds/src/test/rebasing.spec.ts

```diff
@@ -452,15 +452,16 @@ describe("PropertyDDS", () => {
 			});
 			afterEach(async () => {
 				// We expect the internal representation to be the same between both properties
-				expect((sharedPropertyTree1 as any).remoteTipView).to.deep.equal(
-					(sharedPropertyTree2 as any).remoteTipView,
+				expect(((sharedPropertyTree1 as any).remoteTipView as CommitNode).changeSet).to.deep.equal(
```

Comment:
add some type hints to make sure the type checking finds issues that would be hidden by `any`